src/res/flame.html

```diff
@@ -128,13 +366,22 @@ <h1>/*title:*/</h1>
 		}
 	}
 
-	function search(r) {
+	function search(r, resetZoom) {
 		if (r === true && (r = prompt('Enter regexp to search:', '')) === null) {
 			return;
 		}
 
 		pattern = r ? RegExp(r) : undefined;
+
+		if (resetZoom) {
+			root = origRoot;
```

Comment:
What is `origRoot`? Can it be different from the `all` frame at level 0?

Reply:
`origRoot` is the root of the first `search` call, which should be the `all` frame. Renaming to `allRoot`.

Using `levels[0][0]` now.